### ISSUE TITLE
Laravel 5.4 Support

### ISSUE THREAD

**Issue by crishellco** [ixudra/curl#36]
Seems that only one update needs to be made to support Laravel 5.4:

`src/Ixudra/Curl/CurlServiceProvider.php(18)`

```php
$this->app['Curl'] = $this->app->share
```
to 
```php
$this->app['Curl'] = $this->app->singleton
```

**Comment by isaackearl** [comment]
@crishellco this should now be fixed with the latests merge...

**Comment by elimentz** [comment]
Fixed in release 6.7.0
